qcfractal/interface/client.py

```diff
@@ -5,6 +5,10 @@
 import requests
 import yaml
 
+from collections import defaultdict
+
+from .collections import collection_factory
```

Comment:
Well, this is technically correct, albeit a bit confusing. This might be just what we have to do to avoid name collision here.

Reply:
Not optimal, but no way around without renaming collections I believe.

Reply:
I was not suggesting making a change, just noting for myself mostly that this going to be the canonical way we have to handle this